### PR TITLE
chore: update tests to use `toThrow`

### PR DESCRIPTION
**Pull request by iamnewton**
using `toThrow`

### CODE REVIEW

File: test/e2e/interest-groups/index.html [hunk 1]
```diff
@@ -4,7 +4,6 @@
 		<title>interest-groups.test</title>
 	</head>
 	<body>
-		<script src="https://unpkg.com/idb/build/iife/index-min.js"></script>
```
Comment (iamnewton) [inline test/e2e/interest-groups/index.html:7]: new way of `dist` doesn't require this anymore

File: test/e2e/interest-groups/index.test.js [hunk 2]
```diff
@@ -1,52 +1,24 @@
...
-		await expect(await page.title()).toMatch('interest-groups.test');
-	});
-
-	it('should provide the fledge code', async () => {
-		await expect(await page.title()).toMatch('interest-groups.test');
-		const fledge = await page.evaluate(() => window.fledge.fledge);
-		expect(Object.keys(fledge)).toEqual([ 'joinAdInterestGroup', 'leaveAdInterestGroup' ]);
-	});
-});
-
```
Comment (iamnewton) [inline test/e2e/interest-groups/index.test.js:18]: this was moved into a new file to test the API itself on the `window`

File: test/e2e/interest-groups/index.test.js [hunk 3]
```diff
@@ -1,52 +1,24 @@
...
-					name: 'test-interest',
-					bidding_logic_url: 'https://fledge.magnite.com/bl.js',
-				}, 60000),
-			);
-		} catch (e) {
-			errorThrown = true;
-		}
-		expect(errorThrown).toBe(false);
+	it('should return true when provided minimum required params', async () => {
+		const fledge = await page.evaluate(() =>
```
Comment (brodrigu) [inline test/e2e/interest-groups/index.test.js:14]:
```suggestion
		const result = await page.evaluate(() =>
```

File: test/e2e/interest-groups/index.test.js (hunk 2 above)
Comment (brodrigu) [reply test/e2e/interest-groups/index.test.js:18]: you seemed to have forgotten to include the new file in this PR.

File: test/e2e/interest-groups/index.test.js (hunk 3 above)
Comment (brodrigu) [reply test/e2e/interest-groups/index.test.js:14]: `fledge` is not really the right new for the response here.

File: test/e2e/interest-groups/index.html (hunk 1 above)
Comment (brodrigu) [reply test/e2e/interest-groups/index.html:7]: nice, was wondering about that.

**Review by brodrigu — CHANGES_REQUESTED** [review]

File: test/e2e/interest-groups/index.test.js (hunk 2 above)
Comment (iamnewton) [reply test/e2e/interest-groups/index.test.js:18]: yeah, its in the `auction` branch which will eventually be merged in either here or `main`.

File: test/e2e/interest-groups/index.test.js [hunk 4]
```diff
@@ -1,52 +1,24 @@
...
-		expect(errorThrown).toBe(false);
+	it('should return true when provided minimum required params', async () => {
+		const result = await page.evaluate(() =>
+			window.fledge.fledge.joinAdInterestGroup({
+				owner: 'magnite.com',
+				name: 'test-interest',
+				bidding_logic_url: 'https://fledge.magnite.com/bl.js',
+			}, 60000),
+		);
+		expect(fledge).toBe(true);
```
Comment (iamnewton) [inline test/e2e/interest-groups/index.test.js:21]:
```suggestion
		expect(result).toBe(true);
```

**Pull request merged**